tests/internal/test_forksafe.py

```diff
@@ -288,58 +288,61 @@ def fn():
     assert exit_code == 42
 
 
-@pytest.mark.subprocess(
-    out="" if (3,) < sys.version_info < (3, 7) else ("CTCTCT" if sys.platform == "darwin" else "CCCTTT"),
-    err=None,
-    env=dict(_DD_TRACE_GEVENT_HUB_PATCHED="true"),
-)
-def test_gevent_reinit_patch():
-    import os
-    import sys
-
-    from ddtrace.internal import forksafe
-    from ddtrace.internal.periodic import PeriodicService
-
-    class TestService(PeriodicService):
-        def __init__(self):
-            super(TestService, self).__init__(interval=1.0)
+# FIXME: subprocess marks do not respect pytest.mark.skips
```

Comment:
@P403n1x87 I took a quick look but couldn't figure out why the skips weren't working. My hypothesis is that the hook we use in pytest for the subprocess tests runs before the hook that checks for skipping (but haven't been able to confirm)

Reply:
That's probably right. I don't know if there's an easy way to control that. Perhaps we could add a `skip` argument to handle this